bedrock/products/views.py

```diff
@@ -125,7 +121,7 @@ def vpn_pricing_page(request):
 @require_safe
 def vpn_features_page(request):
     template_name = "products/vpn/features.html"
-    ftl_files = ["products/vpn/shared"]
+    ftl_files = ["products/vpn/shared", "products/vpn/features"]
```

Comment:
Issue: we should use `products/vpn/features.ftl` as the activation file, so this should be ordered the other way around:
```suggestion
    ftl_files = ["products/vpn/features", "products/vpn/shared"]
```